S/Singular/build_tarballs.jl

```diff
@@ -27,15 +27,15 @@ import Pkg.Types: VersionSpec
 # to all components.
 #
 name = "Singular"
-upstream_version = v"4.3.1-5" # 4.3.1 plus some changes
-version_offset = v"0.1.0"
+upstream_version = v"4.3.2-0" # 4.3.2p1 plus some changes
+version_offset = v"0.0.0"
```

Comment:
Indeed! My bad, I suggested to @hannes14 that this should be OK (my excuse is that I slept 2 hours last night and my brain is even more mushy than usual)
```suggestion
version_offset = v"0.1.0"
```